testproject/tests/test_second_step_authentication.py

```diff
@@ -572,29 +526,28 @@ def test_change_primary_disabled_method_wrong(active_user):
     response = client.post(
         path="/auth/mfa/change-primary-method/",
         data={
-            "method": "sms",
+            "method": "sms_twilio",
             "code": "code",
         },
         format="json",
     )
-    error_code = "not_enabled"
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too